librosa/core/time_frequency.py

```diff
@@ -844,9 +846,19 @@ def hz_to_octs(frequencies, A440=440.0):
     ----------
     frequencies   : number >0 or np.ndarray [shape=(n,)] or float
         scalar or vector of frequencies
-    A440          : float
+
+    tuning        : number in [-0.5, 0.5)
+        Tuning deviation from A440 in bins per octave.
```

Comment:
given that our default sample rate is 22050 (#509), setting the librosa pitch standard to be A441, not 440, would be nicely consistent. (a 441 Hz tone at 22050 Hz sample rate has exactly 50 samples per period). But it may not satisfy the principle of least surprise. I could have it either way.

Reply:
Hah.  Well, since we're deprecating this parameter anyway, I don't think it much matters, but I'd prefer to keep things in line with [standards](https://en.wikipedia.org/wiki/A440_(pitch_standard)).

Reply:
we're deprecating `A440`, but we're not deprecating the fact that the tuning deviation is computed with respect to 440 Hz, not 441 Hz. But sure, let's keep it in line with ISO 16.